inc/plugins/MybbStuff/MyAlerts/templates/alert_row.html

```diff
@@ -1,24 +1,21 @@
-<tr class="alert-row alert--{$alert['alert_code']} {$alert['alert_status']}"
-    id="alert_row_{$alert['id']}">
-    <td class="{$altbg}" width="50" align="center">
+<tr class="alert {$alert['alert_status']} alert--{$alert['alert_code']}"
+    id="alert_row_popup_{$alert['id']}">
```

Comment:
Should this really be `alert_row_popup_`? Isn't this the template for the full page?

Reply:
You are right, fixed :)